scope & closures/ch2.md

```diff
@@ -129,11 +129,11 @@ Os casos de uso para geração dinâmica de código são incrivelmente raros, vi
 
 ### `with`
 
-The other frowned-upon (and now deprecated!) feature in JavaScript which cheats lexical scope is the `with` keyword. There are multiple valid ways that `with` can be explained, but I will choose here to explain it from the perspective of how it interacts with and affects lexical scope.
+A outra funcionalidade mal vista (e agora desaconselhada!) em JavaScript e com a qual se pode trapacear o escopo léxico é a keyword `with`. Existem muitas maneiras válidas de se explicar `with`, mas vou escolher explicar sob a óptica de como este mecanismo interage e afeta o escopo léxico.
```

Comment:
Que acha da substituição de `keyword` por `palavra-chave`? Como literaturas de outras linguagens, em livros traduzidos, lidam com a palavra? Estou em dúvida.

É um ponto importante essa definição (qualquer que seja), por impactar também `this`,`void`, `return`, etc.